static/script/historian.js

```diff
@@ -18,9 +18,10 @@ define(
              * @constructor
              * @ignore
              */
-            init: function init (currentUrl) {
+            init: function init (currentUrl, UrlAppParams) {
```

Comment:
actually - if currentUrl already contains the query parameters then no need to pass in the UrlAppParams right?

Reply:
This is true - essentially this could be achieved by currentUrl.match(/broadcast=true/) but this seemed far too hacky. Took this approach rather than a) pulling lib/application into historian b) re-implementing the queryParam extraction - but I'm happy with either way really :)